hub/powertoys/run.md

```diff
@@ -157,17 +150,17 @@ If the program plugin's option "Include in global result" is not selected, be su
 
 The Calculator plugin supports the following operations:
 
-| Operation |  Operator Syntax | Description |
-| :- | :- | :- |
-| Addition |  a + b | |
+| Operation | Operator Syntax | Description |
+| :--- | :--- | :--- |
+| Addition | a + b | |
 | Subtraction | a - b | |
 | Multiplication | a * b | |
-| Division |  a / b | |
+| Division | a / b | |
 | Modulo/Remainder | a % b | |
 | Exponentiation | a ^ b | |
-| Ceiling function | ceil( x.y ) |  Rounds a number up to the next larger integer. |
+| Ceiling function | ceil( x.y ) | Rounds a number up to the next larger integer. |
 | Floor function | floor( x.y ) | Rounds a number down to the next smaller integer. |
-| Rounding | round( x.abcd ) | Rounds to the nearest integer.<br />_(Example: `round(8.7867)`)_ |
+| Rounding | round( x.abcd ) | Rounds to the nearest integer. |
```

Comment:
please don't remove.
this should explain that we mean a decimal number input here

Reply:
Debatable. I assume one would know what "rounding" means. We also mention the word "integer" already.

Reply:
With this in mind the example might be to much and it should be clear that `x.abc` means `8.75`.